cache/sharded_cache.cc

```diff
@@ -230,4 +227,26 @@ int ShardedCache::GetNumShardBits() const { return BitsSetToOne(shard_mask_); }
 
 uint32_t ShardedCache::GetNumShards() const { return shard_mask_ + 1; }
 
+inline hash_t ShardedCache::HashSlice(const Slice& s) const {
+  uint64_t in_high64 = static_cast<uint64_t>(*s.data());
```

Comment:
This only pulls out one char (8 bits) and widens to 64 bits. Like `CacheKey::AsSlice()` uses reinterpret_cast on the pointer, we can reinterpret_cast to const uint64_t* here and get two elements.

Reply:
Omg, I forgot to bring my brain to work. Thanks!